oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/apm.yml

```diff
@@ -221,10 +223,23 @@ templates:
                   "metricName": "service_cpm",
                   "queryMetricType": "readMetricsValues",
                   "chartType": "ChartLine",
-                  "unit": "CPM - calls per minute"
+                  "unit": "CPM / PPM",
+                  "tips": "For HTTP 1/2, gRPC, RPC services, this means Calls Per Minute (CPM), for TCP services, this means Packets Per Minute (PPM)"
                 },
                 {
-                  "width": "4",
+                  "width": 3,
+                  "title": "Service Throughput",
```

Comment:
We should be clear this is TCP level metrics, right?

Reply:
> We should be clear this is TCP level metrics, right?

Added tooltip to the metrics